src/odemis/gui/cont/acquisition/cryo_acq.py

```diff
@@ -581,3 +677,13 @@ def _on_settings_va(self, _):
         Called when a VA which might affect the acquisition is modified
         """
         self._update_acquisition_time()
+
+    def _on_features_change(self, features):
+        """ Called when the features are changed.
+        To enable/disable the acquire features button.
+        """
+
+        if features:
+            self._panel.btn_acquire_features.Enable()
+        else:
+            self._panel.btn_acquire_features.Disable()
```

Comment:
Typically, this type of code can be written like this:
```suggestion
        self._panel.btn_acquire_features.Enable(bool(features))
```